main.py

```diff
@@ -1,10 +1,35 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as config_file:
+        players_data = json.load(config_file)
+
+    for player, data in players_data.items():
+
+        race = Race.objects.get_or_create(data["race"]["name"],
+                                          {"name": data["race"]["name"],
+                                           "description": data["race"]["description"]})[0] # noqa
+        if data["guild"]:
+            guild = Guild.objects.get_or_create(data["guild"]["name"],
```

Comment:
Similar to the Race model, the `get_or_create` method for the Guild model should have a field lookup as the first argument, such as `name=data['guild']['name']`.